geospatial/Dockerfile

```diff
@@ -2,29 +2,62 @@ FROM rocker/tidyverse:latest
 MAINTAINER "Carl Boettiger" cboettig@ropensci.org
 
 ENV GDAL_VERSION 2.1.2
+ENV MRAN https://cloud.r-project.org/
 
 ## Add LaTeX, rticles and bookdown support
 RUN echo "deb-src http://deb.debian.org/debian jessie main" >> /etc/apt/sources.list \
   && apt-get update \
 #  && apt-get build-dep -o APT::Get::Build-Dep-Automatic=true libgdal-dev \
   && apt-get install -y --no-install-recommends \
-  netcdf-bin \
-  libgeos-dev \
+  ## Dependencies as used in homebrew gdal2 https://github.com/OSGeo/homebrew-osgeo4mac/blob/master/Formula/gdal2.rb
+  libpng12-dev \
+  libjpeg-dev \
+  libgif-dev \
+  libtiff-dev \
+  libgeotiff-dev \
   libproj-dev \
-  libudunits2-dev \
-  && wget http://download.osgeo.org/gdal/${GDAL_VERSION}/gdal-${GDAL_VERSION}.tar.gz \
-  && tar -xf gdal-${GDAL_VERSION}.tar.gz \
-  && cd gdal* \
-  && ./configure && make && make install \
+  libgeos-dev \
+  libsqlite3-dev \
+  libfreexl.dev \
+  libspatialite-dev \
+  unixodbc-dev \
+  ## Optionsl? Maybe leave out in the base image?
```

Comment:
I'd suggest keeping netcdf; netcdf with the ncdf4 R library provides an interface to both netCDF and HDF5 files.